libs/zzz/page-optimize-pando/src/OptTargetRow/OptSelector.tsx

```diff
@@ -0,0 +1,107 @@
+import { DropdownButton } from '@genshin-optimizer/common/ui'
+import type { TargetTag } from '@genshin-optimizer/zzz/db'
+import {
+  type CharOpt,
+  type ICachedCharacter,
+  targetTag,
+} from '@genshin-optimizer/zzz/db'
+import { useDatabaseContext } from '@genshin-optimizer/zzz/db-ui'
+import { own } from '@genshin-optimizer/zzz/formula'
+import {
+  FullTagDisplay,
+  useZzzCalcContext,
+} from '@genshin-optimizer/zzz/formula-ui'
+import { Box, ListItemText, MenuItem } from '@mui/material'
+import { useMemo } from 'react'
+
+const statTargets = [
```

Comment:
why not just use what comes out of listFormulas?

Reply:
Some of them don't make sense as opt targets